app/src/main/java/com/conupods/OutdoorMaps/CameraController.java

```diff
@@ -65,6 +68,56 @@ public void goToDeviceCurrentLocation() {
         }
     }
 
+
+
+    public void setCurrentLocationCoordinates(LatLng coordinates) {
+        mCurrentLocationCoordinates = coordinates;
```

Comment:
unless you are setting this from outside (which I dont think you should) You should make this private, or even better, delete it

Reply:
made to private, it is used within the getCurrentLocation logic